src/deviceModeInit.js

```diff
@@ -72,7 +72,6 @@ let _rudderTracking = (function () {
       rs$.ajax({
```

Comment:
Additionally, the source config URL should also add the below query parameters to distinguish the requests.

`p` - the platform. Maybe use 'shopify'. For SDKs, we use 'web', 'android', and 'ios' etc.
`v` - version of the source. Optional. Let's add it if we have access to the version.

Feel free to add any other additional relevant query params.

Reply:
do you have any API doc for sourceConfig API ?
https://api.dev.rudderlabs.com/docs#/Source%20Config --> I don't see it supports any query params. I see this change can be done later too

Reply:
We checked in the config backend swagger docs, don't see support for query params in the sourceConfig api mentioned